EmerisHQ/demeris
# fix: added default symbol for pool assets

## Description
1. Performed a check on assets to determine pool assets.
2. Created a GravityIcon icon component for the gravity pool assets.
3. Displayed the GravityIcon if it is a gravity pool asset.

## Testing

1. Go to Portoflio page
2. Click on 'View all' in the assets list
3. Scroll down to 'Gravity 7'
4. See fix
